services/rollout-service/pkg/cmd/server.go

```diff
@@ -66,6 +67,12 @@ type Config struct {
 	RevolutionDoraConcurrency int           `default:"10" split_words:"true"`
 	RevolutionDoraMaxEventAge time.Duration `default:"0" split_words:"true"`
 
+	DataDogDoraEnabled     bool          `split_words:"true"`
```

Comment:
For readability purposes, I would write this as `DatadogDoraEnabled`. Same thing for the other variables